### DISCUSSION TITLE
changer la dénomination "mode brouillon" pour "non répertorié"

### DISCUSSION THREAD

**Discussion by TimotheeLiotard** [EsupPortail/Esup-Pod#1220] [General]
Bonjour, 

Je travaille à l'université Grenoble-alpes en tant qu'ingénieur pédagogique d'un service central d'appui aux enseignants. J'interviens notamment avec l'équipe dont j'ai la charge sur l'utilisation de POD par nos enseignants et personnels administratifs qui ont une utilisation de plus en plus poussée. 

C'est un outil grandement apprécié et qui est devenu indispensable. Cependant je me permet de vous transmettre une difficulté que nous rencontrons à la fois en tant qu'IP ou que l'équipe audiovisuel rencontre avec les enseignants : le terme "mode brouillon" crée beaucoup de confusion parmi nos utilisateurs car il ne leur parait pas suffisamment explicite, ils pensent que cela signifie que leur vidéo est un brouillon et n'est donc pas finalisée. 

Nous avons entrepris un certain nombre d'action (tutoriels, message explicatifs) pour les sensibiliser. Mais même après leur avoir expliqué ils nous font souvent la remarque (et il ne se passe pas une semaine sans que je n'ai) qu'ils trouvent que le nom n'est pas bien adapté, ce à quoi je consens. 

Nous pourrions faire une modification locale dans notre pod ce qui résoudrait la chose à l'UGA. Mais je souhaitai vous proposer l'idée de changer cette appellation à votre niveau pour "non répertorié" (qui ferait écho à la fonctionnalité sur YouTube), car cela pourrait aussi servir pour d'autres utilisateurs, en effet cela me paraitrait surprenant qu'il n'y ait que les personnels/enseignants de l'UGA qui fassent la remarque. En vous remerciant pour l'attention que vous pourrez porter à mon message et encore merci pour tout le travail accompli sur ce projet.
Cordialement,

**Comment by Badatos** [comment]
Bonjour,
Le serveur https://videos.univ-grenoble-alpes.fr utilise la version 3.6.1 de Pod.
Je vous invite à tester la version 3.8, dans laquelle le mode "brouillon" a été renommé "privé / brouillon".
